lib/pushmi_pullyu/cli.rb

```diff
@@ -59,13 +59,15 @@ def configure_rollbar
     Rollbar.configure do |config|
       config.enabled = false unless options[:rollbar][:token].present?
       config.access_token = options[:rollbar][:token]
-
+      config.exception_level_filters.merge!({
+       'IOError' => 'ignore'
+      })
       # add a filter after Rollbar has built the error payload but before it is delivered to the API,
       # in order to strip sensitive information out of certain error messages
       exception_message_transformer = proc do |payload|
-        clean_message = payload[:exception].message.sub(/http:\/\/.+:.+@(.+)\/fedora\/rest\/prod\/(.*)/,
+        clean_message = payload[:exception][:message].sub(/http:\/\/.+:.+@(.+)\/fedora\/rest\/prod\/(.*)/,
```

Comment:
According to Rollbar, the payload json structured like this:
      "exception": {
        "message": "<http://****:*****r@mycombe.library.ualberta.ca:8080/fedora/rest/prod/8c/ff/bb/0f/8cffbb0f-8cb8-4061-b6fe-3122a595d181/list_source>: 404", 
        "class": "IOError"
      }
So we will need to replace the value in payload[:exception][:message] with the scrubbed version.